fixture/auditor.properties.tmpl

```diff
@@ -0,0 +1,160 @@
+#
+# For auditor
+#
```

Comment:
I just copied this `.properties` file from https://github.com/scalar-labs/scalar/blob/v3.9.1/auditor/conf/auditor.properties.tmpl .